### PR TITLE
set Content-Type to "text/plain" for install script

### LINKED ISSUE #3947: https://nixos.org/nix/install should set Content-Type
```
> curl -I https://nixos.org/nix/install
HTTP/2 301 
age: 1741
content-length: 0
date: Wed, 29 Jul 2020 15:16:55 GMT
location: https://releases.nixos.org/nix/nix-2.3.7/install
server: Netlify
via: 1.1 216b2e0a8a27f8fca1b540a1c4ea6922.cloudfront.net (CloudFront)
x-amz-cf-id: YIID5crvOOU7bzvqDROJ7yWS6zFRt7QuhaVsgy_7_uBo-ARR3U-JXw==
x-amz-cf-pop: FRA2-C2
x-cache: Hit from cloudfront
x-nf-request-id: 14aaf732-46fb-4d6e-922d-cfb4808ea41b-15124515
```

There is no `Content-Type` header returned, so clients assume it’s a binary, even though it’s a script.

`Content-Type: text/plain` would be more appropriate.

### PR DESCRIPTION
**Pull request by garbas**
fixes #3947

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nix/3948)

### CODE REVIEW

**Comment by garbas** [comment]
This should now be in the right location (and not https://github.com/NixOS/nixos-channel-scripts/pull/41 as I thought before)

**Comment by garbas** [comment]
I will manually update the content type for current release once this PR gets merged

File: maintainers/upload-release.pl [hunk 1]
```diff
@@ -117,7 +117,16 @@ sub downloadFile {
     my $dstKey = "$releaseDir/" . $name;
     unless (defined $releasesBucket->head_key($dstKey)) {
         print STDERR "uploading $fn to s3://$releasesBucketName/$dstKey...\n";
-        $releasesBucket->add_key_filename($dstKey, $fn)
+
+        my $configuration = ();
```
Comment (edolstra) [inline maintainers/upload-release.pl:121]:
```suggestion
        my $configuration = {};
```

(I think, I didn't test.)

Comment (garbas) [reply maintainers/upload-release.pl:121]: I borrowed this from https://github.com/NixOS/nixos-channel-scripts/blob/master/mirror-nixos-branch.pl#L277

**Pull request merged**